label_studio/io_storages/base_models.py

```diff
@@ -688,12 +688,27 @@ class Meta:
 class ImportStorageLink(models.Model):
 
     task = models.OneToOneField('tasks.Task', on_delete=models.CASCADE, related_name='%(app_label)s_%(class)s')
+
+    # ---
+    # NOTE: ImportStorageLink is task-level, not key-level. These fields could be split out into a new, key-level table to optimize DB access patterns.
+
     key = models.TextField(_('key'), null=False, help_text='External link key')
+
+    # NOTE: unused
```

Comment:
This isn't entirely true, I found a reference to the field here: https://github.com/HumanSignal/label-studio/blob/ade0e61ef9c41cc0941cc04707139c57bf2e1294/label_studio/io_storages/base_models.py#L746

In general I'm not a fan of comments unless they provide essential information; I'm in favor of removing
```suggestion
```

Reply:
Yes, it always gets/creates using the default value of `True`, which is equivalent to the field not existing since it's never accessed. I wrote the comment because I had the question "what's the difference between `exists` and `object_exists`?" - as I think anyone reading this code would, and found that it had a clear answer.

Will remove it if you think it's still out of scope, but this should be documented somewhere, imho, if not here.

Reply:
Something like this then - "unused" could indicate to the developer (incorrectly) that the field doesn't appear in sql queries and is safe to delete, which isn't true.

```suggestion
    # This field is set to True on creation and never updated; it should not be relied upon.
```

Reply:
That's a better wording, thanks. I was relying too heavily on the implicit understanding that we never delete fields anyways :)